### PR TITLE
EWL-5184 -  Theme contact us form

### PR DESCRIPTION
**Pull request by designerneil**
## Ticket(s)

**Jira Ticket**
- [EWL-5184: A1 - Forms Theme contact us form](https://issues.ama-assn.org/browse/EWL-5184)

## Description
Theme the contact us webform in D8

## To Test
- [ ] checkout 
- [ ] switch your local branch of D8 to `feature/EWL-5193-multipage-form`
- [ ] update your local.yml to pull local SG2
- [ ] In your D8 terminal run ` composer install`
- [ ] run `drush @one.local cim -y && drush @one.local cr `
- [ ] visit http://ama-one.local/form/contact-us
- [ ] does it have the same elements and is the `look and feel` broken?
- [ ] Did you test in IE 11?

## Visual Regressions

For reference look at: https://www.ama-assn.org/eform/submit/contact-us
Also refer to: https://americanmedicalassociation.github.io/ama-style-guide-2/?p=pages-form-contact-us

## Relevant Screenshots/GIFs
screen shot 2018-09-28 at 10 39 50 am

## Remaining Tasks
n/a

## Additional Notes
n/a
---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)

### CODE REVIEW

File: styleguide/source/assets/scss/03-organisms/_contact-us-webform.scss [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      display: none;
+    }
+
+    &.ui-state-active {
+      background-color: $purple;
+      color: white;
+    }
+  }
+
+  .form-submit.ama__button {
```
Comment (nstriedinger) [inline styleguide/source/assets/scss/03-organisms/_contact-us-webform.scss]: Couldn't all webform have this button style?

File: styleguide/source/assets/scss/03-organisms/_contact-us-webform.scss [hunk 2]
```diff
@@ -0,0 +1,45 @@
+.webform-submission-contact-us-form {
```
Comment (nstriedinger) [inline styleguide/source/assets/scss/03-organisms/_contact-us-webform.scss:1]: @designerneil Do we need to be so specific?

Comment (nstriedinger) [reply styleguide/source/assets/scss/03-organisms/_contact-us-webform.scss:1]: Might be a good idea to bring this style to all the other webforms.

File: styleguide/source/assets/scss/03-organisms/_contact-us-webform.scss (hunk 1 above)
Comment (designerneil) [reply styleguide/source/assets/scss/03-organisms/_contact-us-webform.scss]: @nstriedinger I assume you're referring to the .ui-checkboxradio-label and form submits?

**Review by nstriedinger — COMMENTED** [review]
@designerneil yup

File: styleguide/source/assets/scss/03-organisms/_contact-us-webform.scss (hunk 1 above)
Comment (nstriedinger) [reply styleguide/source/assets/scss/03-organisms/_contact-us-webform.scss]: yup

**Review by nstriedinger — APPROVED** [review]

**Pull request merged**